spec/controllers/bibliographic_controller_spec.rb

```diff
@@ -415,4 +415,155 @@
       expect(response.body).to include("99122426947506421")
     end
   end
+
+  describe "#availability_holding" do
+    let(:bib_record) { file_fixture("alma/9922486553506421.json") }
+    let(:holding_items) { file_fixture("alma/9922486553506421_holding_items.json") }
+    let(:bib_record_no_holdings) { file_fixture("alma/9922868943506421.json") }
+
+    before do
+      stub_request(:get, "https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs?expand=p_avail,e_avail,d_avail,requests&mms_id=not-exist")
```

Comment:
This is still pending. I can do this next week or we can merge as-is and refactor later.

Reply:
I did take a look at the stub helpers, though. I will need to rename a few of my fixtures so that they work with the helpers but it should not take too long.